client/src/modules/payment/services/payment.ts

```diff
@@ -23,11 +25,13 @@ async function handleApiError (err, defaultMessage = '') {
         message = errorJson.message
       }
     } else if (hasResponseData && responseData instanceof String) {
-      message = responseData
+      message = responseData.toString()
```

Comment:
FYI: this is needed because "message" is automatically declared as a string (not a String) on line 17.